app/weeklyPicks/page.tsx

```diff
@@ -79,31 +79,33 @@ export default function WeeklyPickForm() {
     resolver: zodResolver(FormSchema),
   });
 
-  const onSubmit = useCallback(
-    async (data: z.infer<typeof FormSchema>) => {
-      try {
-        const teamSelect = data.type.toLowerCase();
-        localStorage.setItem('team', data.type);
-
-        const teamID = NFLTeams.find(
-          (team) => team.teamName.toLowerCase() === teamSelect,
-        )?.$id;
-        const resultJSON = `${user.id}:{"team":"${teamID}","correct":true}`;
-        const appendNewResult = allPicks
-          ? `{${allPicks},${resultJSON}}`
-          : `{${resultJSON}}`;
-
-        await createWeeklyPicks({
-          gameId: '66311a210039f0532044',
-          gameWeekId: '6622c7596558b090872b',
-          userResults: appendNewResult,
-        });
-      } catch (error) {
-        console.error('Submission error:', error);
-      }
-    },
-    [NFLTeams, user, allPicks],
-  );
+  const onSubmit = async (data: z.infer<typeof FormSchema>) => {
+    try {
+      const teamSelect = data.type.toLowerCase();
+      localStorage.setItem('team', data.type);
```

Comment:
m: Remove local storage. We dont want to use this at all. We want to grab the data, save it into Zustand, and consume it there.

Reply:
got it! I thought that was in  @vmaineng  next ticket so didn't touch it.  I can make the update :)

Reply:
The state changes are merged into develop :)

Reply:
I got them @vmaineng! :)

Reply:
✅ Done!